integrations/GoogleOptimize/browser.js

```diff
@@ -0,0 +1,66 @@
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class GoogleOptimize {
+  constructor(config) {
+    this.name = "GOOGLE_OPTIMIZE";
+    this.ga = config.ga;
+    this.trackingId = config.trackingId;
+    this.containerId = config.containerId;
+    this.async = config.async;
+    this.aflicker = config.aflicker;
+  }
+
+  init() {
+    logger.debug("===in init Google Optimize===");
+    if (this.ga) {
+      if (!this.trackingId) {
+        return;
+      }
+      ScriptLoader(
+        "Google Tag Manager",
+        `https://www.googletagmanager.com/gtag/js?id=${this.trackingId}`
+      );
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+      gtag("config", `${this.trackingId}`);
+    }
+    if (!this.containerId) {
+      return;
+    }
+    if (this.async) {
+      ScriptLoader(
+        "Google Optimize",
+        `https://www.googleoptimize.com/optimize.js?id=${this.containerId}`,
+        true
+      );
+    } else {
+      ScriptLoader(
+        "Google Optimize",
+        `https://www.googleoptimize.com/optimize.js?id=${this.containerId}`,
+        false
+      );
+    }
+    if (this.aflicker) {
```

Comment:
```suggestion
    if (this.aflicker && this.async) {
```

Reply:
flicker is required only in case of sync that why applying `!this.async`